[debug] [2021-12-01T06:32:28.589Z] ----------------------------------------------------------------------
[debug] [2021-12-01T06:32:28.589Z] Command:       /usr/local/bin/node /usr/local/bin/firebase deploy
[debug] [2021-12-01T06:32:28.590Z] CLI Version:   9.22.0
[debug] [2021-12-01T06:32:28.590Z] Platform:      darwin
[debug] [2021-12-01T06:32:28.590Z] Node Version:  v16.9.1
[debug] [2021-12-01T06:32:28.591Z] Time:          Wed Dec 01 2021 15:32:28 GMT+0900 (대한민국 표준시)
[debug] [2021-12-01T06:32:28.591Z] ----------------------------------------------------------------------
[debug] 
[debug] [2021-12-01T06:32:28.592Z] >>> [apiv2][query] GET https://firebase-public.firebaseio.com/cli.json [none]
[debug] [2021-12-01T06:32:28.623Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2021-12-01T06:32:28.623Z] > authorizing via signed-in user (tjrcjf9@gmail.com)
[debug] [2021-12-01T06:32:28.623Z] [iam] checking project pba-planner for permissions ["firebase.projects.get","firebasehosting.sites.update"]
[debug] [2021-12-01T06:32:28.624Z] > refreshing access token with scopes: ["email","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","openid"]
[debug] [2021-12-01T06:32:28.624Z] >>> HTTP REQUEST POST https://www.googleapis.com/oauth2/v3/token  
 <request body omitted>
[debug] [2021-12-01T06:32:28.883Z] <<< HTTP RESPONSE 200 {"cache-control":"no-cache, no-store, max-age=0, must-revalidate","pragma":"no-cache","date":"Wed, 01 Dec 2021 06:32:28 GMT","expires":"Mon, 01 Jan 1990 00:00:00 GMT","content-type":"application/json; charset=utf-8","vary":"X-Origin, Referer, Origin,Accept-Encoding","server":"scaffolding on HTTPServer2","x-xss-protection":"0","x-frame-options":"SAMEORIGIN","x-content-type-options":"nosniff","alt-svc":"h3=\":443\"; ma=2592000,h3-29=\":443\"; ma=2592000,h3-Q050=\":443\"; ma=2592000,h3-Q046=\":443\"; ma=2592000,h3-Q043=\":443\"; ma=2592000,quic=\":443\"; ma=2592000; v=\"46,43\"","accept-ranges":"none","transfer-encoding":"chunked"}
[debug] [2021-12-01T06:32:28.930Z] >>> HTTP REQUEST POST https://cloudresourcemanager.googleapis.com/v1/projects/pba-planner:testIamPermissions  
 {"permissions":["firebase.projects.get","firebasehosting.sites.update"]}
[debug] [2021-12-01T06:32:29.278Z] <<< [apiv2][status] GET https://firebase-public.firebaseio.com/cli.json 200
[debug] [2021-12-01T06:32:29.279Z] <<< [apiv2][body] GET https://firebase-public.firebaseio.com/cli.json {"cloudBuildErrorAfter":1594252800000,"cloudBuildWarnAfter":1590019200000,"defaultNode10After":1594252800000,"minVersion":"3.0.5","node8DeploysDisabledAfter":1613390400000,"node8RuntimeDisabledAfter":1615809600000,"node8WarnAfter":1600128000000}
[debug] [2021-12-01T06:32:29.991Z] <<< HTTP RESPONSE 200 {"content-type":"application/json; charset=UTF-8","vary":"X-Origin, Referer, Origin,Accept-Encoding","date":"Wed, 01 Dec 2021 06:32:29 GMT","server":"ESF","cache-control":"private","x-xss-protection":"0","x-frame-options":"SAMEORIGIN","x-content-type-options":"nosniff","server-timing":"gfet4t7; dur=764","alt-svc":"h3=\":443\"; ma=2592000,h3-29=\":443\"; ma=2592000,h3-Q050=\":443\"; ma=2592000,h3-Q046=\":443\"; ma=2592000,h3-Q043=\":443\"; ma=2592000,quic=\":443\"; ma=2592000; v=\"46,43\"","accept-ranges":"none","transfer-encoding":"chunked"}
[debug] [2021-12-01T06:32:29.992Z] [iam] error while checking permissions, command may fail: FirebaseError: Authorization failed. This account is missing the following required permissions on project pba-planner:

  firebase.projects.get
  firebasehosting.sites.update
[debug] [2021-12-01T06:32:29.992Z] > refreshing access token with scopes: []
[debug] [2021-12-01T06:32:29.992Z] >>> HTTP REQUEST POST https://www.googleapis.com/oauth2/v3/token  
 <request body omitted>
[debug] [2021-12-01T06:32:30.206Z] <<< HTTP RESPONSE 200 {"expires":"Mon, 01 Jan 1990 00:00:00 GMT","date":"Wed, 01 Dec 2021 06:32:30 GMT","pragma":"no-cache","cache-control":"no-cache, no-store, max-age=0, must-revalidate","content-type":"application/json; charset=utf-8","vary":"X-Origin, Referer, Origin,Accept-Encoding","server":"scaffolding on HTTPServer2","x-xss-protection":"0","x-frame-options":"SAMEORIGIN","x-content-type-options":"nosniff","alt-svc":"h3=\":443\"; ma=2592000,h3-29=\":443\"; ma=2592000,h3-Q050=\":443\"; ma=2592000,h3-Q046=\":443\"; ma=2592000,h3-Q043=\":443\"; ma=2592000,quic=\":443\"; ma=2592000; v=\"46,43\"","accept-ranges":"none","transfer-encoding":"chunked"}
[debug] [2021-12-01T06:32:30.253Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects/pba-planner [none]
[debug] [2021-12-01T06:32:30.830Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects/pba-planner 403
[debug] [2021-12-01T06:32:30.830Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects/pba-planner {"error":{"code":403,"message":"The caller does not have permission","status":"PERMISSION_DENIED"}}
[debug] [2021-12-01T06:32:30.831Z] HTTP Error: 403, The caller does not have permission
[debug] [2021-12-01T06:32:31.065Z] FirebaseError: HTTP Error: 403, The caller does not have permission
    at module.exports (/usr/local/lib/node_modules/firebase-tools/lib/responseToError.js:38:12)
    at Client.doRequest (/usr/local/lib/node_modules/firebase-tools/lib/apiv2.js:241:23)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async Client.request (/usr/local/lib/node_modules/firebase-tools/lib/apiv2.js:96:20)
    at async Object.getFirebaseProject (/usr/local/lib/node_modules/firebase-tools/lib/management/projects.js:290:21)
    at async Object.getDefaultHostingSite (/usr/local/lib/node_modules/firebase-tools/lib/getDefaultHostingSite.js:8:21)
    at async requireHostingSite (/usr/local/lib/node_modules/firebase-tools/lib/requireHostingSite.js:9:18)
    at async Object.fn (/usr/local/lib/node_modules/firebase-tools/lib/commands/deploy.js:73:9)
    at async /usr/local/lib/node_modules/firebase-tools/lib/command.js:188:17
[error] 
[error] Error: Failed to get Firebase project pba-planner. Please make sure the project exists and your account has permission to access it.
